andela/artemis-ah-backend
# #164198498 Follow, Get and Unfollow users

#### What does this PR do?
>- Have the users follow and unfollow each other
>- Have users view who follows them and who they are following

#### Description of Task to be completed?
Having the following endpoint functional:
* GET  `host:port/api/profiles/followers`
* GET  `host:port/api/profiles/following`
* POST `host:port/api/profiles/:username/follow`
* DELETE `hostport/api/profiles/:username/follow`

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Run `npm test` to run the tests to see test results.
4. Run `npm run start:dev` to test manually with Postman.

#### Any background context you want to provide?
* I created a task to capture have users view who follows them and who they are following

#### What are the relevant pivotal tracker stories?
[#164198498](https://pivotaltracker.com/story/show/164198498)
